src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dto/EserviceDTO.java

```diff
@@ -40,37 +23,30 @@
 @ToString
 public class EserviceDTO {
```

Comment:
Use interface

Reply:
this topic was already discussed in the previous pr